litestar/channels/memory.py

```diff
@@ -0,0 +1,77 @@
+from __future__ import annotations
+
+import asyncio
+import contextlib
+from asyncio import Queue
+from collections import defaultdict, deque
+from typing import Any, AsyncGenerator, Iterable
+
+from litestar.channels.base import ChannelsBackend
+
+
+class MemoryChannelsBackend(ChannelsBackend):
```

Comment:
```suggestion
class InMemoryChannelsBackend(ChannelsBackend):
```

Reply:
Why? We also have the `MemoryStore` and not `InMemoryStore`.